.github/workflows/dotnet.yml

```diff
@@ -122,16 +122,16 @@ jobs:
         with:
           dotnet-version: 6.0.x
 
-  e2e_tests:
-    needs: deploy_to_test
-    runs-on: ubuntu-latest
+  # e2e_tests:
```

Comment:
We won't be running any e-2-e tests for the api project, you can just remove it completely

Reply:
Alright, sure